web/src/api/auth/login-callback.ts

```diff
@@ -133,6 +150,93 @@ export const auth0Login = withApiAuthRequired(
         invite_id ? urls.signup({ invite_id }) : urls.signup()
       );
     }
+    let invite: InviteQuery["invite"][number] | null = null;
+
+    if (invite_id) {
```

Comment:
@igorosip0v curious why are we adding this in this PR? I assume we were handling invites in a different place before?